app/routes/dashboards/dashboard.js

```diff
@@ -575,16 +579,20 @@ export default Ember.Route.extend({
                                 parameterName: "sources"
                             },
                             {
-                                parameterPath: ["query", "bool", "filter", 0, "term", "tags.exact"],
-                                parameterName: "tags"
+                                parameterName: "page",
+                                parameterPath: ["from"]
+                            },
+                            {
+                                parameterName: "tags",
+                                parameterPath: ["query", "bool", "filter", 1, "term", "tags"]
```

Comment:
The "sources" parameterPath is `["query", "bool", "filter", 1, "term", "sources"]`, so I think that should be changed to `0` to avoid conflicting paths